src/command_modules/azure-cli-relay/azure/cli/command_modules/relay/__init__.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-import
+# pylint: disable=line-too-long
+
+from azure.cli.core import AzCommandsLoader
+from azure.cli.command_modules.relay._help import helps
+
+
+class relayCommandsLoader(AzCommandsLoader):
+
+    def __init__(self, cli_ctx=None):
+        from azure.cli.core import ModExtensionSuppress
+        from azure.cli.core.commands import CliCommandType
+        relay_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.relay.custom#{}')
+        super(relayCommandsLoader, self).__init__(
+            cli_ctx=cli_ctx, custom_command_type=relay_custom,
+            min_profile='2017-03-10-profile',
+            suppress_extension=ModExtensionSuppress(__name__, 'relay', '0.1.0',
```

Comment:
Do you actually have an extension called relay?  If not, please remove this suppress_extension kwarg.  I didn't see one in the public list of extensions (though maybe you had a private one?)